data-types.md

```diff
@@ -196,32 +199,42 @@ Example of the **Stop Status** object with properties listed:
-| `end_location` | [Point](point) | Required | Location of the end of the trip. See also [Stop-based Geographic Data][stop-based-geo]. |
-| `publication_time` | [Timestamp][ts] | Optional | Date/time that trip became available through the trips endpoint |
-| `fare`                          | [Fare](#fare)                  | Conditionally Required | Fare for the trip (required if trip was completed)             |
-
-(?) Should we keep mix of `trip_` prefix vs. no prefix? E.g. `start_time`.
+| Field                    | Type            | Required/Optional      | Comments |
+| -----                    | ----            | -----------------      | -------- |
+| `provider_id`            | UUID            | Required               | A UUID for the Provider, unique within MDS. See MDS [provider list](/providers.csv). |
+| `data_provider_id`       | UUID            | Optional               | If different than `provider_id`, a UUID for the data solution provider managing this data endpoint. See MDS [provider list](/providers.csv) which includes both service operators and data solution providers. |
+| `device_id`              | UUID            | Required               | A unique device ID in UUID format |
+| `journey_id`             | UUID            | Optional               | A unique [journey ID](/modes#journey-id) for associating collections of trips for its [mode] |
+| `trip_type`              | Enum            | Optional               | **[Mode](/modes#list-of-supported-modes) Specific**. The [trip type](/modes#trip-type) describing the purpose of a trip segment |
+| `trip_id`                | UUID            | Required               | A unique ID for each trip |
+| `trip_duration`          | Integer         | Required               | Time, in Seconds |
+| `trip_distance`          | Integer         | Required               | Trip Distance, in Meters |
+| `trip_attributes`        | Map             | Optional               | **[Mode](/modes#list-of-supported-modes) Specific**. [Trip attributes](/modes#trip-attributes) given as unordered key-value pairs |
+| `start_time`             | [Timestamp][ts] | Required               | Start of the passenger/driver trip |
+| `end_time`               | [Timestamp][ts] | Required               | End of the passenger/driver trip |
+| `start_location`         | [Point](point)  | Required               | Location of the start of the trip. See also [Stop-based Geographic Data][stop-based-geo]. |
+| `end_location`           | [Point](point)  | Required               | Location of the end of the trip. See also [Stop-based Geographic Data][stop-based-geo]. |
+| `publication_time`       | [Timestamp][ts] | Optional               | Date/time that trip became available through the trips endpoint |
+| `reservation_attributes` | [Reservation](#reservation-data) | Required if available | Reservation details, if a reservation initiated this trip
```

Comment:
I think some of these fields are [mode specific trip attributes](https://github.com/openmobilityfoundation/mobility-data-specification/tree/dev/modes#trip-attributes) only, so they would not be here but be in the appropriate mode pages, eg [passenger services](https://github.com/openmobilityfoundation/mobility-data-specification/blob/dev/modes/passenger-services.md#trip-attributes), [car share](https://github.com/openmobilityfoundation/mobility-data-specification/blob/dev/modes/car-share.md#trip-attributes), [delivery robots](https://github.com/openmobilityfoundation/mobility-data-specification/blob/dev/modes/delivery-robots.md#trip-attributes).

There are [others that used to be in the trips object](https://github.com/openmobilityfoundation/mobility-data-specification/tree/dev/provider#trips) that aren't here now, like parking_verification_url, standard_cost, actual_cost, currency.